frontend/src/app/modules/home/organization/create-organization/create-organization.component.html

```diff
@@ -0,0 +1,58 @@
+<p-dialog [(visible)]="display" appendTo="body" [modal]="true" styleClass="new-org">
+    <ng-template pTemplate="header">
+        <span class="header">Create organization</span>
+    </ng-template>
+    <form [formGroup]="formGroup" method="POST">
+        <div class="reg-block">
+            <div class="input-block">
+                <span class="p-float-label">
+                    <input id="float-input-comp-name" type="text" formControlName="organizationName" name="comp-name"
+                        pInputText>
+                    <label for="float-input-comp-name">Organization name</label>
+                </span>
+                <div *ngIf="formGroup.controls.organizationName.touched && formGroup.controls.organizationName.invalid"
```

Comment:
Can we use validaiton with touched and dirty, so user will see if data is not valid with error message before he will remove focus?

Reply:
Fixed